src/adagrams.js

```diff
@@ -1,15 +1,136 @@
 export const drawLetters = () => {
   // Implement this method for wave 1
+  const LETTER_POOL = {
+    A: 9,
+    B: 2,
+    C: 2,
+    D: 4,
+    E: 12,
```

Comment:
Hi Agnes! Great job completing JS Adagrams! Congrats on finishing your first JavaScript project 🎉 Excellent choice with this data structure.